packages/dashboard/src/components/SubwayNav/SubwayNav.styles.tsx

```diff
@@ -0,0 +1,153 @@
+import { ISubwayNavStyles, ISubwayNavStyleProps } from './SubwayNav.types';
+
+const regularFontSize = 14;
+const regularLineHeight = 20;
+const fontWeight = 300;
+const stepIconSize = '16px';
+const subStepIconSize = '8px';
+const commonFontFamily = 'Segoe UI';
+const subwayNavBaseColor = '#333333';
+
+export const getSubwayNavStyles = (props: ISubwayNavStyleProps): ISubwayNavStyles => {
+  const { className } = props;
+
+  return {
+    subwayNavContainer: [
+      {
+        width: '221px',
+        height: '601px',
+        overflow: 'auto',
+        background: 'white'
+      },
+      className
+    ],
+    subwayNavContentContainer: {
+      marginLeft: '48px',
+      marginTop: '48px',
+      marginBottom: '48px'
+    },
+    subwayNavContent: {
+      fontSize: regularFontSize,
+      lineHeight: regularLineHeight,
+      fontWeight: fontWeight,
+      fontFamily: commonFontFamily,
+      color: subwayNavBaseColor,
+      display: 'flex',
+      justifyContent: 'flex-start',
+      flexWrap: 'wrap',
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      wordWrap: 'break-word'
+    },
+    subwayNavStepDiv: {
+      width: '100%'
+    },
+    subwayNavStepConnector: {
+      width: '0px',
+      height: '36px',
+      marginLeft: '7px',
+      border: '1px solid blue'

```

Comment:
>blue [](start = 25, length = 4)

get this from fabric palatte

Reply:
This is just for testing I had added.  I will remove this as we don't need a border.

Reply:
The pattern in other components I saw was this is in the main folder.